blog/2019-05-29-long-blog-post.md

```diff
@@ -5,40 +5,40 @@ authors: endi
 tags: [hello, docusaurus]
 ---
 
-This is the summary of a very long blog post,
+This is the summary of a v\*ry long blog post,
```

Comment:
Where are all these escapes coming from?

Reply:
`very` is a word rejected by remark-simplify, so I'm using an asterisk to hide that naughty word. The backslash is added by Prettier so it's clear that asterisk doesn't delimit a `<em>` section. Not that it really matters since I expect we plan to use Uppy blog files instead of the current ones that are there just as a filler.